src/components/checkout_app/CheckoutApp.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { CheckoutStore } from '../../store';
+
+const CheckoutApp = ({ children }) => {
+  const { state, dispatch } = useContext(CheckoutStore);
+  const { apiPath, token } = state;
+
+  const initCheckout = async () => {
```

Comment:
remember to wrap initCheckout in a useCallback, and add it to the useEffect's dep array, or move initCheckout inside the useEffect

Reply:
Another great callout. I've updated this as well